portfolio/src/main/webapp/index.html

```diff
@@ -6,13 +6,92 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
+  
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
-</html>
+    <!-- Page content -->
+      <div id="content">
+
+
+	<!-- Side navigation -->
+	<div class="sidenav">
+        <div class= "nameintro"> 
+        <h1 style="color:whitesmoke; font-family:Didot;" ><strong> CHISOM OKWOR </strong></h1>
+        <p style="color:whitesmoke; font-family:Didot;" > Welcome to my Portfolio Website </p>
+        </div>
+
+        <div class="menu">
+  		<a href="#home"><font size="-1">HOME </font> </a>
+          <hr />
+  		<a href="#about"> <font size="-1">ABOUT</font> </a>
+          <hr />
+  		<a href="#experience"> <font size="-1">EXPERIENCE </font> </a>
+          <hr />
+        <a href="#portfolio"> <font size="-1">PORTFOLIO </font> </a>
+            <hr />
+  		<a href="#contact"> <font size="-1">CONTACT</font></a>
+          <hr />
+        </div>
+
+	</div>
+
+<div >
```

Comment:
here and below - the indentation looks out of whack? you want the indentation level to be correct so that it's easier to read. many IDEs offer some sort of auto formatting, or there are tools online as well I'm sure.

Reply:
Alright